packages/@okta/vuepress-site/docs/guides/request-user-consent/main/index.md

```diff
@@ -160,36 +172,51 @@ This example shows the JSON body for a PUT request to the default Custom Authori
 }
 ```
 
+<ApiLifecycle access="ea" />
+To update scope consent to `OPTIONAL`, set `consent` to `REQUIRED` and include the `optional` parameter set to `true`:
+
+```json
+{
+    "id": "scpixa2zmc8Eumvjb0h7",
+    "name": "phone",
+    "displayName": "phone",
+    "description": "Allows this application to access your phone number.",
+    "system": true,
+    "metadataPublish": "ALL_CLIENTS",
+    "consent": "REQUIRED",
+    "optional": true,
+    "default": false
+}
+```
+
 ## Build the request
 
 After you define the scopes that you want to require consent for, prepare an authentication or authorization request with the correct values.
 
-1. Obtain the following values from your OpenID Connect application, both of which can be found on the application's **General** tab:
+1. Obtain the following values from your OpenID Connect application, both of which you can find on the application's **General** tab:
 
     * Client ID
     * Redirect URI
 
-2. Use the default Custom Authorization Server's authorization endpoint:
-
-    > **Note:** See [Authorization Servers](/docs/guides/customize-authz-server/overview/) for more information on the types of authorization servers available to you and what you can use them for.
+2. Use the default custom authorization server's authorization endpoint:
 
-    A default Custom Authorization endpoint looks like this where the `${authorizationServerId}` is `default`:
+    > **Note:** See [Authorization servers](/docs/guides/customize-authz-server/overview/) for more information on the types of authorization servers available to you and what you can use them for.
```

Comment:
I'd move the **Note:** underneath the endpoint, which allows you to remove some of the verbiage that currently comes after, ie., "A default custom authorization endpoint looks like this ..." The Step action takes care of that info. If you think it necessary, you can leave in the detail that `default` replaces `${authorizationServerId}` in the endpoint. Hope this makes sense.